templates/footer_info.twig

```diff
@@ -3,9 +3,9 @@
     <p>2C Hayes Road, Rosebery NSW, 2018, Australia</p>
   </div>
   <div class="info">
-    <p>Copyright © Greenpeace Australia Pacific 2023</p>
+    <p>Copyright © Greenpeace Australia Pacific 2024</p>
```

Comment:
Do we want this to always be 2024? Or should it just be the current year? 
If the latter we could have this auto-update.

Reply:
Current year would be better.

Reply:
```suggestion
    <p>Copyright © Greenpeace Australia Pacific {{ "now"|date("Y") }}</p>
```

This should achieve that.